### PR TITLE
Document rebuilding app after mounting storage

### PR DESCRIPTION
**Pull request by slava-vishnyakov**
It took me quite a while to find how to restart an app.

### CODE REVIEW

File: docs/advanced-usage/persistent-storage.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ dokku storage:mount app-name /var/lib/dokku/data/storage:/storage
 
 Dokku will then mount the shared contents of `/var/lib/dokku/data/storage` to `/storage` inside the container.
 
+You will also need to restart the deployment, to activate storage:
+
+```shell
+dokku deploy app-name
```
Comment (josegonzalez) [inline docs/advanced-usage/persistent-storage.md]: Should be `dokku ps:rebuild APP`, and a reference to the [ps docs](http://dokku.viewdocs.io/dokku/deployment/process-management/).

File: docs/deployment/application-deployment.md [hunk 2]
```diff
@@ -101,6 +101,15 @@ If you only want to rebuild and tag a container, you can skip the deployment pha
 dokku config:set ruby-rails-sample DOKKU_SKIP_DEPLOY=true
 ```
 
+### Re-Deploying / restarting
+
+If you need to re-deploy (or restart) your app:
+
+```shell
+# on your Dokku host
+dokku deploy ruby-rails-sample
```
Comment (josegonzalez) [inline docs/deployment/application-deployment.md]: Should be `dokku ps:rebuild APP`, and a reference to the ps docs.

File: docs/advanced-usage/persistent-storage.md (hunk 1 above)
Comment (slava-vishnyakov) [inline docs/advanced-usage/persistent-storage.md]: Sure! Changed in both places. I'm not sure how to refer to docs, so just wrote (see ps) with link.

File: docs/advanced-usage/persistent-storage.md [hunk 3]
```diff
@@ -25,6 +25,12 @@ dokku storage:mount app-name /var/lib/dokku/data/storage:/storage
 
 Dokku will then mount the shared contents of `/var/lib/dokku/data/storage` to `/storage` inside the container.
 
+You will also need to restart the deployment, to activate storage (see [ps](http://dokku.viewdocs.io/dokku/deployment/process-management/)):
```
Comment (josegonzalez) [inline docs/advanced-usage/persistent-storage.md]:
```
Once you have mounted persistent storage, you will also need to restart the application. See the [process scaling documentation](/dokku/deployment/process-management) for more information.
```

Comment (slava-vishnyakov) [inline docs/advanced-usage/persistent-storage.md]: :+1:

File: docs/deployment/application-deployment.md [hunk 4]
```diff
@@ -101,6 +101,17 @@ If you only want to rebuild and tag a container, you can skip the deployment pha
...
+### Re-Deploying / restarting
+
+If you need to re-deploy (or restart) your app: 
+
+```shell
+# on your Dokku host
+dokku ps:rebuild ruby-rails-sample
+```
+
+(See the [process scaling documentation](/dokku/deployment/process-management) for more information.
```
Comment (josegonzalez) [inline docs/deployment/application-deployment.md]: There is an extra parens here.

Comment (slava-vishnyakov) [inline docs/deployment/application-deployment.md]: Yeah, sorry :) fixed already.

**Comment by josegonzalez** [comment]
Thanks for the pull request (and revisions)!

**Comment by slava-vishnyakov** [comment]
Thanks for the corrections 👍  :)

**Pull request merged**